docs/source/config_files.rst

```diff
@@ -1,9 +1,32 @@
 .. _config-files:
 
-Supported configuration files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Configuration Files
+~~~~~~~~~~~~~~~~~~~
 
-Below is a list of supported configuration files (roughly in the order of build priority).
+``repo2docker`` looks for configuration files in the repository being built
```

Comment:
... in the root directory and the `binder/` sub-directory... ? I think in the config file section of the docs we should mention all the things. Maybe in the introduction/walk through we only mention one way?

Reply:
I only mention it 1 way because that's the way the example repo works :/ IMHO I think most workflows will not specify a `binder` folder even though we prioritize that folder

Reply:
Ah, I thought this was the "config files reference" section of the documentation.